PZ/WebClient/models.py

```diff
@@ -6,3 +6,9 @@ class Monitors(models.Model):
     monitor_name = models.CharField(max_length=200)
     monitor_domain = models.URLField()
     user_id = models.ForeignKey(User)
+
+class Hosts(models.Model):
+    monitor_id_id = models.IntegerField()
+    host_name = models.CharField(max_length=200)

```

Comment:
Names are local for classes, so I don't think `host_` before variable names are necessary, what do you think @PatrykMatwijczuk ?